vertical-pod-autoscaler/pkg/recommender/logic/recommender_test.go

```diff
@@ -63,3 +63,29 @@ func TestMinResourcesSplitAcrossContainers(t *testing.T) {
 	assert.Equal(t, model.MemoryAmountFromBytes((*podMinMemoryMb*1024*1024)/2), recommendedResources["container-2"].Target[model.ResourceMemory])
 	assert.Equal(t, model.MemoryAmountFromBytes((*podMinMemoryMb*1024*1024)/2), recommendedResources["container-2"].Target[model.ResourceMemory])
 }
+
+func TestControlledResourcesFiltered(t *testing.T) {
```

Comment:
Can we have one more test for [CPU, memory]? To make sure default works as expected.

Reply:
Done.